src/scripts/win_start.bat

```diff
@@ -0,0 +1,54 @@
+@echo off
+setlocal enabledelayedexpansion
+
+set RELEASE_ROOT=%~dp0\..
+
+if not defined RELEASE_NAME (set RELEASE_NAME=example_cli_app)
```

Comment:
Does it make sense for us to use this as a default here, or should we just be exiting with an error?

Reply:
Another option, possibly for a future change, would be to use `eex` templates for these scripts, and then render them using the release name as a parameter, so that we don't need to rely on setting the variable when we build the wrapper.

Reply:
+1 to the eex idea -- I'll roll that into a future PR

Reply:
updated to EEx!